Training1/pipeline_hazards/.cproject

```diff
@@ -21,6 +21,7 @@
 							<tool id="legup.managedbuild.tool.gnu.cpp.compiler.exe.debug.1129551615" name="GCC C++ Compiler" superClass="legup.managedbuild.tool.gnu.cpp.compiler.exe.debug">
 								<option id="gnu.cpp.compiler.option.optimization.level.2042635163" name="Optimization Level" superClass="gnu.cpp.compiler.option.optimization.level" useByScannerDiscovery="false" value="gnu.cpp.compiler.optimization.level.most" valueType="enumerated"/>
 								<option id="gnu.cpp.compiler.option.debugging.level.1787542767" name="Debug Level" superClass="gnu.cpp.compiler.option.debugging.level" useByScannerDiscovery="false" value="gnu.cpp.compiler.debugging.level.none" valueType="enumerated"/>
+								<option id="gnu.cpp.compiler.option.dialect.std.2033344015" superClass="gnu.cpp.compiler.option.dialect.std" useByScannerDiscovery="true" value="gnu.cpp.compiler.dialect.c++11" valueType="enumerated"/>

```

Comment:
I think I mentioned in Teams yesterday that the ID has to be unique in each .cproject yesterday right? Now that I take a closer look it looks like you just copy the line without changing the number?

For testing, you can see what happens when you import 2 of these projects in the same IDE session. I think to be safe you should just make sure these numbers are different

Reply:
I tried importing 2 projects on Linux and Windows. The same suffix ID didn't affect anything.